hack/devtools/local_dev_env.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# Local development environment script.
+# Execute this script from the root folder of the repo (ARO-RP).
+# This script is aimed to provide an automatic and easy way to prepare 
+# the environment and execute the ARO RP locally.
+# The steps here are the ones defined in docs/deploy-development-rp.md
+# We recommend to use this script after you understand the steps of the process, not before.
+
+
+build_development_az_aro_extension() {
+    echo "INFO: Building development az aro extension..."
+    make az
+}
+
+verify_aro_extension() {
+    echo "INFO: Verifying aro extension..."
+    grep -q 'dev_sources' ~/.azure/config || cat >>~/.azure/config <<EOF
+[extension]
+dev_sources = $PWD/python
+EOF
+}
+
+set_storage_account() {
+    echo "INFO: Setting storage account..."
+    export SECRET_SA_ACCOUNT_NAME=rharosecretsdev make secrets
+}
+
+apply_default_env_config() {
+    echo "INFO: Applying default environment config file..."
+    cat >env <<EOF
+export LOCATION=eastus
+export ARO_IMAGE=arointsvc.azurecr.io/aro:latest
+export RP_MODE=development # to use a development RP running at https://localhost:8443/
```

Comment:
What happens if the script fails after setting these values? shouldn't we undo the changes?

Reply:
As those are just environment variables, I would say that there is no action needed to undo the changes if something fails later. 

With the new warning (mentioned above) the user knows if he/she is overwritten something in the configuration.